microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -391,6 +420,33 @@ Kids Mode does not collect data for personalization of the news feed or other Mi
 Microsoft Edge WebDriver allows developers to drive the Microsoft Edge browser using the [WebDriver protocol](https://www.w3.org/TR/webdriver2/).  The Microsoft Edge WebDriver is an executable file `msedgedriver.exe` separate from Microsoft Edge. Developers can call the driver from  client code, such as a test script.  By default, Microsoft Edge WebDriver sends diagnostic data such as the status of the [New Session WebDriver command](https://www.w3.org/TR/webdriver2/#new-session) to Microsoft.  To turn off the diagnostic data collection for Microsoft Edge WebDriver, set the `MSEDGEDRIVER_TELEMETRY_OPTOUT` environment variable to `1`.  For more information about Microsoft Edge WebDriver, see [Use WebDriver to automate Microsoft Edge automation](/microsoft-edge/webdriver-chromium).
 
 
+<!-- ====================================================================== -->
+## Rewards
+
+Microsoft Rewards (the "Program") enables you to earn redeemable points for activities such as qualified searches, acquisitions, and other offers from Microsoft. These points can then be redeemed for gift cards or non-profit donations or automatic contributions to subscriptions. Microsoft Edge has built-in support for all existing avenues for earning points and also introduces new and exclusive means to earn more than ever. You can easily track your points, stay up to date with limited period offers to get the most out of the program.
+For non-members, Microsoft Edge presents personalized joining offers based on the page you’re on. Once you engage and choose to enroll, Edge uses your Microsoft account identity to sign you up and start the rewards service that automatically adds points to your account for searching, playing and shopping on Edge. Edge does not send your personal information or browsing data to any third-party services. 
```

Comment:
```suggestion

For non-members, Microsoft Edge presents personalized joining offers based on the page you're on. Once you engage and choose to enroll, Microsoft Edge uses your Microsoft account identity to sign you up and start the rewards service.  The rewards service automatically adds points to your account for searching, playing and shopping on Microsoft Edge. Microsoft Edge doesn't send your personal information or browsing data to any third-party services. 
```
* separate the paragraphs via blank line.
* prefer contractions.
* this Docs repo strictly puts Microsoft before Edge per mgmt.